docs/tools/CLI/build_profiles.md

```diff
@@ -1,42 +1,42 @@
 <h2 id="build-profiles">Build profiles</h2>
 
-Arm Mbed OS 5 supports three primary build profiles: *develop*, *debug* and *release*. The Online Compiler uses the *develop* profile. When building from Arm Mbed CLI, you can select a profile by adding the `--profile <profile>` flag. You can specify custom user-defined profiles by giving the path to the profile.
+Arm Mbed OS 5 defines three collections of toolchain flags used during the build. These are __build profiles__. These are collections of toolchain flags used during the build. The three build profiles are *develop*, *debug* and *release*. The Mbed Online Compiler uses the *develop* build profile. When building from Arm Mbed CLI, you may select the __build profile__ by passing your desired build profile, by name or path, to the __--profile__ argument.
```

Comment:
Remove the redundant sentence, use code format for code.

```suggestion
Arm Mbed OS 5 defines three collections of toolchain flags used during the build. These are __build profiles__.  The three build profiles are *develop*, *debug* and *release*. The Mbed Online Compiler uses the *develop* build profile. When building from Arm Mbed CLI, you may select the __build profile__ by passing your desired build profile, by name or path, to the `--profile` argument.
```

Reply:
@AnotherButler - What do you think of @theotherjimmy suggestion? Wanted to check with you as you had that modified little bit.

Reply:
@SenRamakri I'm fine with this, but I can't click the "commit suggestion" button. I think there's a bug. Feel free to do so yourself, or I can make the changes to your PR manually.

Reply:
@AnotherButler - Thanks for confirming, and I'll make the modifications along with other review comments fixes.